### ISSUE TITLE
Ability to change the shimmer colour?

### ISSUE THREAD

**Issue by Reedyuk** [facebook/Shimmer#39]
At the moment the shimmer takes the colour from the object its shimmering, it would be nice if you could specify a colour for the shimmer to be.
An example, a white label, could have a gold shimmer. 
I hope that makes sense.

Thanks
Andrew.

**Comment by facebook-github-bot-5** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by osolano** [comment]
I'd like to request this feature as well. For me, if the color of the object is green or blue and then shimmer is white. This can make something have a shiny object look effect.

**Comment by osolano** [comment]
Nevermind. I figured it out thank you for this effect!

**Comment by jyounus** [comment]
Can you please share what you did to achieve that @osolano?

**Comment by aehlke** [comment]
@jyounus @osolano I'd also like to know :)

**Comment by jarnoverreyt** [comment]
@grp ,

Is there a solution?

@osolano @jyounus ,

Can you tell us how you fixed it?

cc\ @aehlke

**Comment by jyounus** [comment]
I wasn't able to change the colour @jarnoverreyt, only make it brighter or darker by playing around with these two properties:

```
shimmeringAnimationOpacity
shimmeringOpacity
```

**Comment by jarnoverreyt** [comment]
@jyounus ,

Thanks for the quick answer but playing around with these two properties didn't do the trick for me.. It has to be a lot darker, but because my light background, it stays light..

**Comment by jyounus** [comment]
Yeah, unfortunately I had to change the design slightly for a specific screen because of that. :/

**Comment by ravelasquez99** [comment]
Did anyone figure out how to change the color?

**Comment by FareedQ** [comment]
Looking at the code behind, it doesn't seem like it does this. It just fades the opacity in an area. This isn't a feature it currently does. :(

**Comment by FareedQ** [comment]
A trick to do this, is have another view behind it that is the colour you want. Then when it fades the opacity it will reveal the colour you want to shine.
